test/backend/routes.spec.js

```diff
@@ -695,129 +742,566 @@ describe('Routes:', () => {
-                            [ 'us_ag_exports_2011', 'fruits fresh', 'numeric' ],
-                            [ 'usa_states_2014', 'rank', 'integer' ],
-                            [ 'usa_states_2014', 'pop', 'numeric' ],
-                            [ 'usa_states_2014', 'state', 'character varying' ],
-                            [ 'usa_states_2014', 'postal', 'character varying' ],
-                            [ 'us_cities_2014', 'name', 'character varying' ],
-                            [ 'us_cities_2014', 'lon', 'numeric' ],
-                            [ 'us_cities_2014', 'pop', 'integer' ],
-                            [ 'us_cities_2014', 'lat', 'numeric' ],
-                            [ 'walmart_store_openings_1962_2006', 'opendate', 'character varying' ],
-                            [ 'walmart_store_openings_1962_2006', 'year', 'integer' ],
-                            [ 'walmart_store_openings_1962_2006', 'day', 'integer' ],
-                            [ 'walmart_store_openings_1962_2006', 'month', 'integer' ],
-                            [ 'walmart_store_openings_1962_2006', 'lon', 'numeric' ],
-                            [ 'walmart_store_openings_1962_2006', 'lat', 'numeric' ],
-                            [ 'walmart_store_openings_1962_2006', 'type_store', 'character varying' ],
-                            [ 'walmart_store_openings_1962_2006', 'zipcode', 'integer' ],
-                            [ 'walmart_store_openings_1962_2006', 'strstate', 'character varying' ],
-                            [ 'walmart_store_openings_1962_2006', 'strcity', 'character varying' ],
-                            [ 'walmart_store_openings_1962_2006', 'streetaddr', 'character varying' ],
-                            [ 'walmart_store_openings_1962_2006', 'county', 'integer' ],
-                            [ 'walmart_store_openings_1962_2006', 'st', 'integer' ],
-                            [ 'walmart_store_openings_1962_2006', 'conversion', 'character varying' ],
-                            [ 'walmart_store_openings_1962_2006', 'date_super', 'character varying' ],
-                            [ 'walmart_store_openings_1962_2006', '1', 'integer' ],
-                            [ 'weather_data_seattle_2016', 'min_temperaturec', 'character varying' ],
-                            [ 'weather_data_seattle_2016', 'mean_temperaturec', 'character varying' ],
-                            [ 'weather_data_seattle_2016', 'max_temperaturec', 'character varying' ],
-                            [ 'weather_data_seattle_2016', 'date', 'character varying' ],
-                            [ 'world_gdp_with_codes_2014', 'code', 'character varying' ],
-                            [ 'world_gdp_with_codes_2014', 'gdp (billions)', 'numeric' ],
-                            [ 'world_gdp_with_codes_2014', 'country', 'character varying' ]
+                            [ 'my_new_topo."edge"', 'edge_id', 'integer' ],
```

Comment:
Entering the large data in tests here is not good.. Can you please open an issue to move these large, static datasets to a separate file.. `spec_constant.js` or something?